packages/core/lib/src/tweet/widgets/likes.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:core/src/styles/app_colors.dart';
+import 'package:core/src/tweet/widgets/like_icon_button.dart';
+import 'package:core/src/tweet/widgets/metric_text.dart';
+import 'package:flutter/material.dart';
+
+/// Tweet likes icon and metric text widget
+class Likes extends StatefulWidget {
+  /// Creates new instance of [Likes]
+  const Likes({
+    super.key,
+    this.metricValue = 0,
+    this.iconSize = 15,
+  });
```

Comment:
Do we need to have a `onChanged` `VoidCallback` property here? How do we trigger changes to our backend?